src/TuDa.CIMS.Shared/Entities/AssetItem.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace TuDa.CIMS.Shared.Entities;
+
+/// <summary>
+/// An entity representing an item in the chemical inventory.
+/// </summary>
+public record AssetItem()
```

Comment:
🌱 This should be abstract, so only sub records can be actually created
```suggestion
public abstract record AssetItem
```